src/stringProcessing/precedenceExceptionWithoutRegex.js

```diff
@@ -0,0 +1,54 @@
+import { get, includes } from "lodash-es";
```

Comment:
This file has such a nice implementation now compared to the mess it was before. Also super nice, extensive comments 🙇‍♂️